sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -64,6 +67,13 @@ class CachedTableSuite extends QueryTest with SQLTestUtils with SharedSQLContext
     maybeBlock.nonEmpty
   }
 
+  def isExpectStorageLevel(rddId: Int, level: DataReadMethod.DataReadMethod): Boolean = {
```

Comment:
`DataReadMethod.DataReadMethod` -> `DataReadMethod` and 
remove line 25, `import org.apache.spark.executor.DataReadMethod`.